engine/collection/pusher/engine.go

```diff
@@ -144,7 +138,7 @@ func (e *Engine) SubmitCollectionGuarantee(guarantee *flow.CollectionGuarantee)
 	// network usage significantly by implementing a simple retry mechanism here and
 	// only sending to a single consensus node.
 	// => https://github.com/dapperlabs/flow-go/issues/4358
```

Comment:
better to update this comment to avoid confusing

Reply:
https://github.com/onflow/flow-go/pull/1406/commits/9d45cd47ff0a911c4ddaffaaceec94e74cbbdee6